mms/tests/integration_tests/test_utils.py

```diff
@@ -0,0 +1,122 @@
+                _download_file(download_dir, URL[onnx_model])
+                _download_file(download_dir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/signature.json")
+                _download_file(download_dir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/synset.txt")
+                model_tar = '{}/{}.tar.gz'.format(download_dir,onnx_model) 
+                tar = tarfile.open(model_tar, "r:*")
+                tar.extractall(path=download_dir)
+                tar.close()
+                model_dir = '{}/{}'.format(download_dir, onnx_model)
+                model_path = os.path.join(model_dir, 'model.onnx')
+                new_path = os.path.join(model_dir, '{}.onnx'.format(onnx_model))
+                os.rename(model_path, new_path)
+                shutil.move(new_path, download_dir)
+       
+            else:
+                _download_file(download_dir, URL[onnx_model])
+                _download_file(download_dir, "https://s3.amazonaws.com/model-server/models/" + onnx_model + "/signature.json")
+                _download_file(download_dir, "https://s3.amazonaws.com/model-server/models/" + onnx_model + "/synset.txt")
+            print('model files prepared for model {} '.format(onnx_model))
+            # Export the model.
+            print("Exporting the mxnet model server model...")
+            subprocess.check_call(['mxnet-model-export', '--model-name', onnx_model, '--model-path', download_dir], cwd=download_dir)
+            shutil.move('{}/{}.model'.format(download_dir, onnx_model), tmpdir)
+            for root, dirs, files in os.walk(download_dir):
+                for f in files:
+                    os.unlink(os.path.join(root, f))
+                for d in dirs:
+                    shutil.rmtree(os.path.join(root, d))
+            shutil.rmtree(download_dir)
+        else:
+            _download_file(tmpdir, URL[onnx_model]) 
+    except Exception as e:
+        print("Failed to create models. {}".format(str(e)))
+        raise
+  
+
+
+def start_test(tmpdir, URL, onnx_model, port = '8080', onnx_source_model_zoo = False, is_onnx_model = True, test_multiple_models = False):
+   model_list = []
+   model_names = []
+   if test_multiple_models:
+        for model in URL.keys():
+            create_model(tmpdir, URL, model, onnx_source_model_zoo, is_onnx_model)
+            model_names.append( '{}={}/{}.model'.format(model, tmpdir, model))
+            model_list.append(model)
+   else:
+        model_names.append('{}={}/{}.model'.format(onnx_model, tmpdir, onnx_model))
+        create_model(tmpdir, URL, onnx_model, onnx_source_model_zoo, is_onnx_model) 
+        model_list.append(onnx_model)    
+   server_pid = subprocess.Popen(['mxnet-model-server', '--models']+ model_names+[ '--port',port]).pid
+   
+    #server_pid = subprocess.Popen(['mxnet-model-server', '--models', '{}={}/{}.model'.format(onnx_model, tmpdir, onnx_model),'--port',port]).pid
+   try:  
+        time.sleep(50)
```

Comment:
do we need 50s? if we are mostly testing 1 model at a time, and multi-model test(s) are just a few simple models, this can be lower.

Reply:
@jesterhazy The tests were failing with 30. So, I increased it

Reply:
@jesterhazy  I can make it 30 by testing smaller 3-4 models in multi model test. Will do that